homekit/protocol/feature_flags.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 Robert Schulze
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from enum import IntEnum
+
+
+class FeatureFlags(IntEnum):
```

Comment:
Could we combine this file with homekit/model/feature_flags.py to prevent to files named the same in different places?

Reply:
I would also propose to move model/feature_flags to protocol, since e.g. protocol/status_codes then follows the same implementation pattern.